.config/forest.dic

```diff
@@ -42,29 +46,41 @@ mainnet
 MDNS
 mempool
 Merkle
+METADATA_FILE_SUFFIX
 MiB
 milliGas
 multihash
 multisig
 mutex
 P2P
 parsable
+parse_filename
 pointer/SM
 PoSt
+recognised
```

Comment:
Let's just use the US version in the documentation. We are using EN-US dictionary.
```suggestion
```